samples/storageclass/powerstore-replication.yaml

```diff
@@ -19,7 +19,7 @@ kind: StorageClass
 metadata:
   name: "powerstore-replication"
 provisioner: "csi-powerstore.dellemc.com"
-reclaimPolicy: Retain
+reclaimPolicy: Delete
 volumeBindingMode: Immediate
 parameters:
   # replicationPrefix paramater in values.yaml must be used as prefix for all replication parameters in storage class
```

Comment:
Does the replicationPrefix still apply since this is not a generic SC but one geared for Metro?

Reply:
This prefix is used for replicator side-car deployment which is not required for Metro, but we reused the same prefix in SC parameters.